internal/services/fingerprint/consumer_test.go

```diff
@@ -176,7 +176,7 @@ func (s *ConsumerTestSuite) TestVinCredentialerHandler_DeviceFingerprint() {
 			VehicleNFT:        nft,
 			AftermarketDevice: aftermarketDevice,
 		},
-		{
+		/* {
```

Comment:
Merge in main when you get the chance. We're just going to skip this until we can remove the dependency on the net.